client/i18n/locales/english/intro.json

```diff
@@ -2060,6 +2060,13 @@
       "front-end-development-certification-exam": {
         "title": "Front End Development Certification Exam",
         "intro": ["", ""]
+      },
+      "workshop-loan-qualification-checker": {
```

Comment:
Also, this will need to be moved in the correct spot.
According to the sheet this should be block 154.
So replace this 

```md
 "nnqa": { "title": "154", "intro": [] },
```
with the intro text for the loan qualification checker

Reply:
This will still need to be moved to the correct order

Reply:
the goal was to replace ` "nnqa": { "title": "154", "intro": [] },` with the intro text you wrote for the loan checker.
but it looks like you already resolved that